c++/src/kj/compat/http.c++

```diff
@@ -5716,14 +5716,111 @@ public:
     if (!connectSettings.useTls) {
```

Comment:
I notice this is still wrapped in `#if KJ_HAS_OPENSSL`. We broke that dependency with abstraction, so you can remove that `#if`.